keps/sig-node/3386-kubelet-evented-pleg/README.md

```diff
@@ -348,6 +356,31 @@ We expect no non-infra related flakes in the last month as a GA graduation crite
 - Add E2E Node Conformance presubmit job in CI 
 - Add E2E Node Conformance periodic job in CI
 
```

Comment:
If Generic PLEG is used as a fallback mechanism, we should have a graduation criteria to enable it by default to have all tests running against Generic PLEG continuously in CI environment.